enzyme/Enzyme/CMakeLists.txt

```diff
@@ -208,6 +208,13 @@ target_compile_options(ClangEnzymeFlags INTERFACE "-fplugin=$<TARGET_FILE:ClangE
 add_custom_target(ClangEnzymeDummy "" DEPENDS ClangEnzyme-${LLVM_VERSION_MAJOR})
 add_dependencies(ClangEnzymeFlags ClangEnzymeDummy)
 
+set_property(TARGET ClangEnzyme-${LLVM_VERSION_MAJOR}
+    PROPERTY PUBLIC_HEADER
+    ${PROJECT_SOURCE_DIR}/include/enzyme/enzyme
```

Comment:
also can we add a warning to the top of the file that changes in these installed files will not change the versions used by enzyme (which are internal to the compiler)

Reply:
> to the top of the file

you mean the top of the line or the top of the include files?

Reply:
to the top of the include files